WalletWasabi/WabiSabi/Client/ArenaResponse.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using System.Linq;
+using WalletWasabi.Crypto.ZeroKnowledge;
+
+namespace WalletWasabi.WabiSabi.Client
+{
+	public class ArenaResponse
+	{
+		public ArenaResponse(IEnumerable<Credential> realAmountCredentials, IEnumerable<Credential> realVsizeCredentials)
+		{
+			RealAmountCredentials = realAmountCredentials.ToArray();
+			RealVsizeCredentials = realVsizeCredentials.ToArray();
+		}
+		public Credential[] RealAmountCredentials { get; }
+		public Credential[] RealVsizeCredentials { get; }
+	}
+
+	public class ArenaResponse<T> : ArenaResponse
```

Comment:
I am not sure if moving one of this classes makes any sense but in case we decide to enforce the one-class-one-file rule, how should the files be called? @molnard any suggestion?

Reply:
It is fine. It just likes having an optional ctor.